content/en/docs/application-store/built-in-apps/radondb-postgresql-app.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Deploy RadonDB PostgreSQL on KubeSphere"
+keywords: 'KubeSphere, Kubernetes, Installation, RadonDB PostgreSQL'
+description: 'Learn how to deploy RadonDB PostgreSQL from the App Store of KubeSphere and access its service.'
+linkTitle: "Deploy RadonDB PostgreSQL on KubeSphere"
+weight: 14294
+---
+
+[RadonDB PostgreSQL](https://github.com/radondb/radondb-postgresql-kubernetes) is an open source, cloud-native, and highly available cluster solution based on [PostgreSQL](https://postgresql.org) database system.
+
+This tutorial demonstrates how to deploy RadonDB PostgreSQL from the App Store of KubeSphere.
+
+## Prerequisites
+
+- Please make sure you [enable the OpenPitrix system](../../../pluggable-components/app-store/).
+- You need to create a workspace, a project, and a user account (`project-regular`) for this tutorial. The account needs to be a platform regular user and to be invited as the project operator with the `operator` role. In this tutorial, you log in as `project-regular` and work in the project `demo-project` in the workspace `demo-workspace`. For more information, see [Create Workspaces, Projects, Accounts and Roles](../../../quick-start/create-workspace-and-project/).
+
+## Hands-on Lab
+
+### Step 1: Deploy RadonDB PostgreSQL from the App Store
+
+1. On the **Overview** page of the project `demo-project`, click **App Store** in the top-left corner.
```

Comment:
`top-left` -> 'upper-left`
KS Doc team decided to use `upper-left/right` instead of `top-left/right`.

Reply:
done